test/button_test.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-env node, mocha */
+
+const assert = require("assert");
+const utils = require("./utils");
+const clipboardy = require("clipboardy");
+// const webdriver = require("selenium-webdriver");
+
+// TODO these may be useful for more complex tests
+// const firefox = require("selenium-webdriver/firefox");
+// const Context = firefox.Context;
+
+// TODO create new profile per test?
+// then we can test with a clean profile every time
+
+describe("Add-on Functional Tests", function() {
+  // This gives Firefox time to start, and us a bit longer during some of the tests.
+  this.timeout(10000);
+
+  let driver;
+  let addonId;
+
+  before(async() => {
+    const newDriver = await utils.promiseSetupDriver();
+    driver = newDriver;
+    return Promise.resolve();
```

Comment:
Not sure. I think this is a remnant of the Promise version of the code that I only partially converted to async/await.